diff -up ./Makefile.rh ./Makefile
--- ./Makefile.rh	2019-09-26 11:18:26.536441290 -0400
+++ ./Makefile	2019-09-26 11:20:13.485118687 -0400
@@ -1,6 +1,7 @@
 # Compile Noise Source as user space application
 
 CC ?= gcc
+INSTALL ?= install
 CFLAGS +=-Wextra -Wall -pedantic -fPIC -O0
 #Hardening
 CFLAGS +=-fstack-protector-all -fwrapv --param ssp-buffer-size=4
@@ -56,14 +57,13 @@ cppcheck:
 	cppcheck --force -q --enable=performance --enable=warning --enable=portability *.h *.c
 
 install:
-	install -d -m 0755 $(DESTDIR)$(PREFIX)/share/man/man3
-	install -m 644 doc/$(NAME).3 $(DESTDIR)$(PREFIX)/share/man/man3/
+	$(INSTALL) -d -m 0755 $(DESTDIR)$(PREFIX)/share/man/man3
+	$(INSTALL) -m 644 doc/$(NAME).3 $(DESTDIR)$(PREFIX)/share/man/man3/
 	gzip -9 $(DESTDIR)$(PREFIX)/share/man/man3/$(NAME).3
-	install -d -m 0755 $(DESTDIR)$(PREFIX)/$(LIBDIR)
-	install -m 0755 -s lib$(NAME).so.$(LIBVERSION) $(DESTDIR)$(PREFIX)/$(LIBDIR)/
-	install -d -m 0755 $(DESTDIR)$(PREFIX)/$(INCDIR)
-	install -m 0644 jitterentropy.h $(DESTDIR)$(PREFIX)/$(INCDIR)/
-	install -m 0644 jitterentropy-base-user.h $(DESTDIR)$(PREFIX)/$(INCDIR)/
+	$(INSTALL) -d -m 0755 $(DESTDIR)$(PREFIX)/$(LIBDIR)
+	$(INSTALL) -m 0755 lib$(NAME).so.$(LIBVERSION) $(DESTDIR)$(PREFIX)/$(LIBDIR)/
+	$(INSTALL) -m 0644 jitterentropy.h $(DESTDIR)$(PREFIX)/$(INCDIR)/
+	$(INSTALL) -m 0644 jitterentropy-base-user.h $(DESTDIR)$(PREFIX)/$(INCDIR)/
 	$(RM) $(DESTDIR)$(PREFIX)/$(LIBDIR)/lib$(NAME).so.$(LIBMAJOR)
 	ln -s lib$(NAME).so.$(LIBVERSION) $(DESTDIR)$(PREFIX)/$(LIBDIR)/lib$(NAME).so.$(LIBMAJOR)
 	ln -s lib$(NAME).so.$(LIBMAJOR) $(DESTDIR)$(PREFIX)/$(LIBDIR)/lib$(NAME).so
